### DISCUSSION TITLE
Welcome to zombienet-sdk Discussions!

### DISCUSSION THREAD

**Discussion by wirednkod** [paritytech/zombienet-sdk#19] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 0xsecaas** [comment]
Hey everyone,  
I’ve recently started exploring the Polkadot ecosystem. I’m especially interested in learning about the potential of DAOs. To get there, I first need to understand the basics. While experimenting, I tried Zombienet but found the setup cumbersome compared to tools like Anvil or the Foundry toolset. That led me to discover Zombienet-SDK.  

Looking forward to learning more.
